packages/controlled-form/src/lib/Checkbox.tsx

```diff
@@ -1,37 +1,49 @@
 import { Checkbox, CheckboxProps } from '@availity/mui-checkbox';
-import { useFormContext, RegisterOptions, FieldValues } from 'react-hook-form';
+import { RegisterOptions, FieldValues, Controller, ControllerProps } from 'react-hook-form';
 
-export type ControlledCheckboxProps = CheckboxProps & {
-  name: string;
-} & Omit<
+export type ControlledCheckboxProps = CheckboxProps &
+  Omit<
     RegisterOptions<FieldValues, string>,
-    'required' | 'max' | 'maxLength' | 'min' | 'minLength' | 'pattern' | 'validate'
-  >;
+    | 'required'
+    | 'disabled'
+    | 'valueAsNumber'
+    | 'valueAsDate'
+    | 'setValueAs'
+    | 'max'
+    | 'maxLength'
+    | 'min'
+    | 'minLength'
+    | 'pattern'
+  > &
+  Pick<ControllerProps, 'defaultValue' | 'shouldUnregister' | 'name'>;
 
 export const ControlledCheckbox = ({
   name,
-  setValueAs,
   disabled,
   onChange,
   onBlur,
   value,
+  defaultValue = false,
```

Comment:
This is needed to be consistent with previous usage. I also feel like it is nice to have... maybe all of the components should have a default defaultValue (typically an empty string) to avoid the case where no value is provided and undefined is used... which then triggers the "changing from uncontrolled to controlled" react message.